### ISSUE TITLE
🐞 Bug report: Elementor Pro - JavaScript not compatible with Safari 14 on iPhone due to usage of static keyword.

### ISSUE THREAD

**Issue by rtpHarry** [elementor/elementor#18090]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I've been looking into an issue this evening and found out that Elementor Pro doesn't support my testing rig of IOS 14 / Safari 14 / iPhone 8.

I appreciate the iPhone 8 itself is very old but IOS 14 only came out in Sept 2020. I don't think it's unreasonable to support it.

Perhaps this is just an issue with the transpiler used?

The issue is that you use the `static` keyword in your code.

I was getting an error with the minified version, so I swapped out the unminified version to get a better debug view.

<img width="1056" alt="image" src="https://user-images.githubusercontent.com/1038062/157125594-1549efbf-c523-4b6a-92fd-aefbcc75eef5.png">

With the minified version it just shows the exact same message, except for saying its line:2.

Looking at the snippet it is failing on:

```JavaScript
static symbolsContainer;
```

<img width="750" alt="image" src="https://user-images.githubusercontent.com/1038062/157125679-edbadca8-f944-48dc-a87a-f274dbc8529b.png">

And looking on caniuse.com this has poor support on Safari 14 IOS and desktop:

https://caniuse.com/mdn-javascript_classes_static_class_fields

<img width="1018" alt="image" src="https://user-images.githubusercontent.com/1038062/157126099-ae47ff3e-5962-4483-a171-f5b5db35a8e0.png">

<img width="835" alt="image" src="https://user-images.githubusercontent.com/1038062/157126216-70f4edb1-6e5f-45e5-a16d-8a627e587402.png">

Is there some kind of tweak that can be made to support this browser please?

With this code in place it is triggering a JavaScript error and breaking the rest of the page. My slideshow doesn't work, the hamburger menu won't open. It means the site is totally unusable on IOS14.

### Steps to reproduce

  1. Open a site that uses Elementor Pro inside a version of Safari that doesn't support the static keyword

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

```
					
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v5.7.37
	PHP Version: 7.4.28
	PHP Max Input Vars: 1000
	PHP Max Post Size: 64M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.9.1
	Site URL: https://mclarenexcell.com
	Home URL: https://mclarenexcell.com
	WP Multisite: No
	Max Upload Size: 64 MB
	Memory limit: 1024M
	Permalink Structure: /%postname%/
	Language: en-GB
	Timezone: Europe/London
	Admin Email: matthew@sokastudio.co.uk
	Debug Mode: Inactive

== Theme ==
	Name: Mclaren Excell
	Version: 2.3.1.1615501073
	Author: Child Theme Configurator
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.5.0
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.109 Safari/537.36

== Active Plugins ==
	Activity Log
		Version: 2.8.2
		Author: Activity Log Team

	Cachebuster
		Version: 1.3
		Author: Albert Peschar

	Child Theme Configurator
		Version: 2.5.8
		Author: Lilaea Media

	Current Year Shortcode
		Version: 1.1.0
		Author: Matthew Harris, runthings.dev

	Disable User Login
		Version: 1.3.2
		Author: Saint Systems

	Duplicate Page
		Version: 4.4.8
		Author: mndpsingh287

	Duplicator Pro
		Version: 4.5.0
		Author: Snap Creek

	Elementor
		Version: 3.5.6
		Author: Elementor.com

	Elementor Pro
		Version: 3.6.3
		Author: Elementor.com

	EWWW Image Optimizer
		Version: 6.4.1
		Author: Exactly WWW

	GDPR Cookie Consent
		Version: 2.1.1
		Author: WebToffee

	Infinite Scroll Elementor
		Version: 2.3.2
		Author: Joy Chetry

	JetElements For Elementor
		Version: 2.6.2
		Author: Crocoblock

	JetEngine
		Version: 2.11.4
		Author: Crocoblock

	JetMenu
		Version: 2.1.6
		Author: Crocoblock

	Mailgun
		Version: 1.7.9
		Author: Mailgun

	MainWP Child
		Version: 4.1.10
		Author: MainWP

	Really Simple SSL
		Version: 5.2.3
		Author: Really Simple Plugins

	Redirection
		Version: 5.2.3
		Author: John Godley

	Slider Revolution
		Version: 6.4.3
		Author: ThemePunch

	Snazzy Maps
		Version: 1.4.0
		Author: Snazzy Maps

	SOKA Disable Media Title
		Version: 1.0.0
		Author: Matthew Harris

	SOKA Elementor Image Carousel Gallery ID
		Version: 1.1.0
		Author: Matthew Harris, SOKA Studio

	Unlimited Elements for Elementor (Premium)
		Version: 1.5.4
		Author: Unlimited Elements

	Wordfence Security
		Version: 7.5.8
		Author: Wordfence

	WP Google Maps
		Version: 8.1.21
		Author: WP Google Maps

	Yoast SEO
		Version: 18.2
		Author: Team Yoast


== Elements Usage ==
	
	footer : 1
		image : 1
		text-editor : 4
		theme-site-logo : 1
	header : 3
		image : 3
		jet-mobile-menu : 2
		menu-anchor : 2
		nav-menu : 2
	page : 5
		button : 2
		heading : 8
		image : 18
		image-carousel : 1
		jet-listing-grid : 3
		slider_revolution : 1
		text-editor : 16
		toggle : 2
	popup : 1
		html : 1
		image : 1
		slides : 1
	section : 9
		image : 13
		nav-menu : 1
	wp-post : 0
		button : 34
		heading : 97
		image : 564
		image-carousel : 66
		jet-listing-grid : 34
		text-editor : 35
		theme-post-title : 1
		toggle : 30
	wp-page : 11
		button : 5
		form : 1
		heading : 7
		html : 1
		image : 8
		image-carousel : 18
		jet-listing-grid : 3
		slider_revolution : 1
		text-editor : 105
		ucaddon_blox_grid_instagrm_widget : 2
		video : 1
		wp-widget-nav_menu : 2
	single-post : 2
		button : 2
		heading : 6
		image : 16
		image-carousel : 2
		text-editor : 2
		theme-post-title : 2
		toggle : 2
		ucaddon_post_grid : 2


== Elementor Experiments ==
	Optimised DOM Output: Active by default
	Improved asset loading: Inactive by default
	Improved CSS Loading: Inactive by default
	Inline Font Icons: Inactive by default
	Accessibility improvements: Active by default
	Import Export Template Kit: Active by default
	Additional Custom Breakpoints: Inactive by default
	Hide native WordPress widgets from search results: Active by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Inactive by default
	Landing pages: Active by default
	Color Sampler: Active by default
	Favorite Widgets: Active by default
	Admin Top Bar: Active by default
	Page Transitions: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default


== Log ==
	
Log: showing 20 of 382022-02-09 17:50:04 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.6.0',
  'to' => '3.6.1',
)]
2022-02-09 17:50:05 [info] elementor-pro::elementor_pro_updater Started 
2022-02-09 17:50:05 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-02-09 17:50:05 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-02-09 17:50:05 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.6.0',
  'to' => '3.6.1',
)]
2022-02-14 22:47:19 [info] elementor-pro::elementor_pro_updater Started 
2022-02-14 22:47:19 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-02-14 22:47:19 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-02-14 22:47:19 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.6.1',
  'to' => '3.6.2',
)]
2022-02-14 22:47:19 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.6.1',
  'to' => '3.6.2',
)]
2022-02-28 17:07:48 [info] elementor-pro::elementor_pro_updater Started 
2022-02-28 17:07:48 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-02-28 17:07:48 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-02-28 17:07:48 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.6.2',
  'to' => '3.6.3',
)]
2022-02-28 17:07:48 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.6.2',
  'to' => '3.6.3',
)]
2022-03-01 13:20:18 [info] elementor::elementor_updater Started 
2022-03-01 13:20:18 [info] Elementor/Upgrades - _on_each_version Start  
2022-03-01 13:20:18 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.5.5',
  'to' => '3.5.6',
)]
2022-03-01 13:20:20 [info] Elementor/Upgrades - _on_each_version Finished 
2022-03-01 13:20:20 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.5.5',
  'to' => '3.5.6',
)]

PHP: showing 4 of 4PHP: 2022-02-21 13:00:08 [notice X 2][/home/mclarenexcell/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-02-24 16:57:06 [notice X 1][/home/mclarenexcell/public_html/wp-content/plugins/elementor-pro/license/updater.php::84] Undefined index: new_version [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-02-28 17:24:50 [notice X 76][/home/mclarenexcell/public_html/wp-content/plugins/elementor-pro/modules/forms/classes/form-record.php::179] Undefined index: referrer [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-03-02 14:02:51 [notice X 1][/home/mclarenexcell/public_html/wp-content/plugins/elementor-pro/modules/forms/classes/recaptcha-v3-handler.php::124] Undefined index: score [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified
	Infinite Scroll Elementor: Compatibility not specified
	JetElements For Elementor: Compatibility not specified
	SOKA Elementor Image Carousel Gallery ID: Compatibility not specified
	Unlimited Elements for Elementor (Premium): Compatibility not specified

== Elementor Pro - Compatibility Tag ==
	

```

**Comment by rtpHarry** [comment]
FYI if you need to do this for your site you will need to do two additional things:

1. Contact Elementor support to get them to send you 3.6.2 as the rollback in the plugin doesn't have anything above 3.5.2 at the time of writing
2. Go to the plugins page and make sure autoupdate is turned off for that plugin. Something went wrong with the previous few versions which was triggering this and it might still be enabled. It caught me out and updated itself again right after I had given the client the all-clear.

**Comment by rtpHarry** [comment]
This PR should fix the issue, I will reopen it if it doesn't.

https://github.com/elementor/elementor/pull/18276

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor Pro v3.6.5 & Elementor v3.6.2

Feel free to update

Thanks!
